Source/SwiftLintFramework/Rules/ModifiersOrderRule.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  ModifiersOrderRule.swift
+//  SwiftLint
+//
+//  Created by Jose Cheyo Jimenez on 05/06/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+// TODO: Get somehow `SwiftDeclarationAttribute` for Swift 4.1
+public enum SwiftDeclarationAttributeKind: String, SwiftLangSyntax  {
+    case `ibaction`         = "source.decl.attribute.ibaction"
+    case `iboutlet`         = "source.decl.attribute.iboutlet"
+    case `ibdesignable`     = "source.decl.attribute.ibdesignable"
+    case `ibinspectable`    = "source.decl.attribute.ibinspectable"
+    case `objc`             = "source.decl.attribute.objc"
+    case name               = "source.decl.attribute.objc.name"
+    case available          = "source.decl.attribute.available"
+    case `final`            = "source.decl.attribute.final"
+    case `required`         = "source.decl.attribute.required"
+    case `optional`         = "source.decl.attribute.optional"
+    case noreturn           = "source.decl.attribute.noreturn"
+    case `NSManaged`        = "source.decl.attribute.NSManaged"
+    case `lazy`             = "source.decl.attribute.lazy"
+    case `dynamic`          = "source.decl.attribute.dynamic"
+    case infix              = "source.decl.attribute.infix"
+    case prefix             = "source.decl.attribute.prefix"
+    case postfix            = "source.decl.attribute.postfix"
+    case autoclosure        = "source.decl.attribute.autoclosure"
+    case noescape           = "source.decl.attribute.noescape"
+    case nonobjc            = "source.decl.attribute.nonobjc"
+    case objcMembers        = "source.decl.attribute.objcMembers"
+    case `mutating`         = "source.decl.attribute.mutating"
+    case `nonmutating`      = "source.decl.attribute.nonmutating"
+    case `convenience`      = "source.decl.attribute.convenience"
+    case `override`         = "source.decl.attribute.override"
+    case `weak`             = "source.decl.attribute.weak"
+    case `testable`         = "source.decl.attribute.testable"
+    case privateSetter      = "source.decl.attribute.setter_access.private"
```

Comment:
Is there more options like public and internal open etc

Reply:
There must be, but yesterday i was not able to grep the values from `SwiftDeclarationAttribute `. I only have the 9.3 beta, and i can not install it as my default, i still need 9.2 for work. And as far i have seen `SourceKitService` framework is only available with your default Xcode. I will try to figure something out...